CHANGELOG.md

```diff
@@ -7,15 +7,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Added
+
+- A new class, `XrpUtils` contains utility functions for XRP specific actions. Previously, this functionality was available via the `Utils` class.
+
+### Deprecated
+
+- XRP specific functionality on the `Utils` class is deprecated. Please use the new `XrpUtils` class instead.
```

Comment:
- XRP-specific functionality on the `Utils` class is deprecated. Use the new `XrpUtils` class instead.